public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -116,7 +118,7 @@ function FolderGroupAndEvaluationInterval({
 
   return (
     <div>
-      <FolderAndGroup groupfoldersForGrafana={groupfoldersForGrafana?.result} />
+      <FolderAndGroup groupfoldersForGrafana={groupfoldersForGrafana?.result} includeProvisioned={includeProvisioned} />
```

Comment:
Should we rename `includeProvisioned` to something like `enableProvisionedGroups`?
The reason for this change is that provisioned groups are always included (they are visible) but are disabled and thus not selectable

Reply:
you're right!